### ISSUE TITLE
dataset and implemeneting unpool

### ISSUE THREAD

**Issue by han-yeol** [CaelynCheung1996/QuickNAT_tensorflow_V2#4]
is OASIS data 3D data?

I have used your unpool function.
but it didn't working.
On second thought, my dataset is 2d image.

**Comment by CaelynCheung1996** [comment]
> is OASIS data 3D data?
> 
> I have used your unpool function.
> but it didn't working.
> On second thought, my dataset is 2d image.

Yes. OASIS data is 3D but the network actually works on 2D slices.
